### DISCUSSION TITLE
/nice-article/csdn/tengxbyfplmdlydbkcsdnbk.html

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#79] [Announcements]
# /nice-article/csdn/tengxbyfplmdlydbkcsdnbk.html

互联网裁员浪潮下，我从腾讯毕业了。2022 年 6 月中旬收到通知，到 8 月初离职，再到 9 月底入职新公司，三个多月转瞬即逝。趁国庆在家时间充裕，现简单地回顾“毕业之旅”。

https://tobebetterjavaer.com/nice-article/csdn/tengxbyfplmdlydbkcsdnbk.html

**Comment by JLQmiller** [comment]
请问大佬你面铂石，面完提交材料以后的流程处理的快么?
